recipes/hyperbox-brain/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "hyperbox-brain" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41ba26a8b285f48ef5f8e76c6f60fdab43b9ed689b7f2c531e7ade4442a83437
```

```suggestion
  url: https://github.com/UTS-CASLab/hyperbox-brain/archive/refs/tags/v{{ version }}.tar.gz
  sha256: 69c49bba5afbadc5c5dcda92b02b0b9a200a93481b5ea6c6f78307d924d5926b
```
Use the GitHub release so that the LICENSE file is pulled from the source file instead of manually including it with the recipe. This way, the license will always be up to date.

---

@carterbox  Thanks for your suggestion. It has been done now.